server/routes/creatingLicences/handlers/additionalLicenceConditionUploadsHandler.ts

```diff
@@ -11,7 +11,8 @@ export default class AdditionalLicenceConditionUploadsHandler {
   GET = async (req: Request, res: Response): Promise<void> => {
     const { licenceId, conditionCode } = req.params
     const { user } = res.locals
-    const licence = await this.licenceService.getLicence(licenceId, user)
+    // TODO: move to get licence from res.locals?
+    const licence = await this.licenceService.getLicence(parseInt(licenceId, 10), user)
```

Comment:
I've highlighted a few places I think licence can be taken from res.locals... will look at in follow up